widgets/qt_view_laser_mode.py

```diff
@@ -0,0 +1,54 @@
+import sys
+from PyQt5.QtWidgets import *
+from PyQt5.QtCore import *
+from PyQt5.QtGui import *
+import qt_custom_decorations
+
+
+class InitializeComboButton(QWidget):
```

Comment:
also i believe we can name this class better

Reply:
I named custom_decorations and custom_sliders with the intention of there being multiple custom classes in each file. I don't think I am quite clear on the one class per file, as it forces a double naming structure that feels a bit clunky. Maybe you can explain a good way to to this when we chat.

Reply:
sure, we can chat about it.